spec/service_providers/webhooks_create_spec.rb

```diff
@@ -207,6 +207,31 @@
     end
   end
 
+  context "when consumer is specified using a label" do
+    before do
+      params.delete(:consumer)
+      params.delete(:provider)
+      params.merge!(consumer_label: "consumer_label")
+      request_body["consumer"] = { "label" => "consumer_label" }
+      mock_pact_broker_index(self)
+
+      pact_broker
+        .given("the 'Pricing Service' and 'Condor' already exist in the pact-broker")
```

Comment:
I don't think we need a provider state, because the label doesn't require any pacticipants to exist already.

We should also expect the body that we sent in the request to be returned in the response, otherwise we can fall into the "garbage in, garbage out" trap https://docs.pact.io/consumer#beware-of-garbage-in-garbage-out-with-putpostpatch The original example is lacking!